src/GraphQL.Tests/Types/AllScalarGraphTypeTests.cs

```diff
@@ -575,13 +575,14 @@ public void toAST_ok(Type graphType, object value)
                 BigInteger bi => new GraphQLIntValue(bi),
                 float f => new GraphQLFloatValue(f),
                 double d => new GraphQLFloatValue(d),
-                _ => null
+                _ => throw new InvalidOperationException("Invalid value type")
```

Comment:
This is a test.  The exception only exists because it won't compile without a `_ => ...` clause.  The exception will never hit (or the test is broken).

Reply:
but i changed it anyway...